### ISSUE TITLE
Deprecation warning due to invalid escape sequences.

### ISSUE THREAD

**Issue by tirkarthi** [demurgos/py-pathmatch#2]
Deprecation warning due to invalid escape sequences. Using raw strings or escaping them again helps in resolving this. Check https://github.com/asottile/pyupgrade/ for automatic fix of this.

```
find . -iname '*.py' | grep -Ev 'rdf4|doc|benchmark|tool' | xargs -P4 -I{} python3.8 -Wall -m py_compile {}
./pathmatch/wildmatch.py:52: DeprecationWarning: invalid escape sequence \A
  _PY_IMPOSSIBLE_MATCH = u'(?:a\A)'
```

**Comment by demurgos** [comment]
Thanks for the issue!
I am no longer using this library but will update it.

**Comment by demurgos** [comment]
Closed by d84fdc52effbd48bfa5bec5e5215bdda73146fc1
